ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -299,7 +303,7 @@ export default defineComponent({
       ),
       phoneNumberRules: customRules(
         required('Enter a phone number'),
-        maxLength(15)
```

Comment:
No maxLength for the phone? Or masking will limit the number of digits the user can input?

Reply:
Masking will limit the number of digits.